com.lunarclient.LunarClient.yml

```diff
@@ -0,0 +1,69 @@
+app-id: com.lunarclient.LunarClient
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+branch: stable
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+command: lunarclient
+separate-locales: false
+rename-icon: lunarclient
+finish-args:
+  - --persist=.minecraft
+  - --persist=.lunarclient
+  - --share=network
+  - --device=dri
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --persist=.java
+modules:
+  - name: unappimage
+    buildsystem: simple
+    build-commands:
+      - make -C squashfs-tools install INSTALL_DIR=/app/bin
+    sources:
+      - type: git
+        url: https://github.com/refi64/unappimage
+        commit: d7f86f2a0d7ec3a69211125207d5f127386b849a
+
+  - name: lunarclient
+    buildsystem: simple
+    cleanup:
+      - squashfs-root
+    sources:
+      - type: extra-data
```

Comment:
I still suggest you ask upstream for redistribution rights.

Reply:
@phase which is one Lunar devs already left a "hooray" reaction on the main PR comment so I guess they're fine with it. It would be definitely good if they could leave a comment instead here though.

Reply:
> I still suggest you ask upstream for redistribution rights.

Why?

Reply:
> > I still suggest you ask upstream for redistribution rights.
> 
> Why?

I think faster downloads, not having broken links and delta updates are the main reasons for the flathub CDN being preffered

Reply:
I do not think it is a problem currently.

The flatpak works & downloads fine.

Reply:
"Fine" is really a poor evaluation here. By stubbornly sticking to extra-data, it's losing delta updates and a proper CDN setup for faster downloads. It's great you're happy with it but it could be done better if you could simply clarify that with upstream.

Reply:
I sent your request to the developers.

Reply:
No reply on that, I do not want this holding up users from using this application that is working